### PR TITLE
Add command-line switch for setting FBFCs in RK32

### PR DESCRIPTION
**Pull request by jeremy-lilly**
@dengwirda, this adds an optional command line switch that sets the value for `[beta1, beta2, beta3]` in RK32-FB for quick testing. 

Ex:
```
python3 swe.py ... --integrate RK32-FB --betas 0.5 0.5 0.33
```

The default value is `1./3 1./2 89./300`.

### CODE REVIEW

**Comment by dengwirda** [comment]
Thanks Jeremy, it makes a lot of sense to add this for the parameter tuning you're doing. I'll tag a few suggestions through the code though.

File: _dt.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def step_RK22(mesh, trsk, flow, cnfg, hh_cell, uu_edge):
            rv_dual, pv_dual, ke_bias, pv_bias
 
 
-def step_RK32(mesh, trsk, flow, cnfg, hh_cell, uu_edge):
+def step_RK32(mesh, trsk, flow, cnfg, hh_cell, uu_edge, betas):
```
Comment (dengwirda) [inline _dt.py]: Do you mind if we put this in the `cnfg` object --- just to keep all of the user-defined things together.

File: swe.py [hunk 2]
```diff
@@ -536,6 +537,14 @@ def init_file(name, cnfg, save):
         required=False, 
         default=False, help="Disable coriolis terms.")
 
+    parser.add_argument(
+        "--betas", dest="betas", type=float,
```
Comment (dengwirda) [inline swe.py]: I wonder if there's a way that we could do this in a general way that might extend to other FB integration schemes in the future? [This post](https://stackoverflow.com/questions/20165843/argparse-how-to-handle-variable-number-of-arguments-nargs) suggests that it's possible to setup arbitrary length array inputs in the argparser, so we could use this for `RK22-FB` as well, for example. The question then is just how to handle the defaults --- what about if this array is default empty in the argparser, and only overrides the hard-coded values in the `step_RKXX` routines when it's explicitly user-defined?
How about also a slight name change to `--FB-weight`, just to make it a bit more descriptive?

**Comment by dengwirda** [comment]
Just to add a few other FB parameter choices too:
- `(0.41250000, 0.43333333, 0.36250000)` has a 1d-wave `alpha_max = 4.67` or so, but is quite bad for nonlinear cases.
- `(0.38999999, 0.45000000, 0.34444444)` has a 1d-wave `alpha_max = 4.00` and seems more of an intermediate choice.

These are both just random choices from the brute-force search, so not necessarily optimal by any stretch.

File: _dt.py (hunk 1 above)
Comment (jeremy-lilly) [reply _dt.py]: Don't mind at all! I wasn't paying attention and didn't even see that `cnfg` was passed in. See https://github.com/dengwirda/swe-python/pull/4/commits/8ec2febb0ecf6a3b6b309aa7f3181a682395b9de.

File: swe.py (hunk 2 above)
Comment (jeremy-lilly) [reply swe.py]: I think this is a great idea -- see https://github.com/dengwirda/swe-python/pull/4/commits/273eef573eb4eb614b2844a76ac9560a65421796. Here, we just check if `FB-weight` was set, if so we use the provided values, if not use the hard-coded ones. This applies to both RK22 and RK32.

https://github.com/dengwirda/swe-python/pull/4/commits/eec48202234d11ca91942ad75e651f3f8d17a10e adds a check to make sure that if `FB-weight` is set, the correct number of weights for either RK22 or RK32 were provided.

The variable name change was done in https://github.com/dengwirda/swe-python/pull/4/commits/f5991f99951ffc5a4ced3e7ac48b20cf28764a5e.

**Review by dengwirda — APPROVED** [review]
Thanks Jeremy!

**Pull request merged**